tests/foreman/api/test_docker.py

```diff
@@ -1370,7 +1372,14 @@ def test_positive_delete(self):
         @Assert: The docker containers are deleted in local and external
         compute resources
         """
-        for compute_resource in (self.cr_internal, self.cr_external):
+        cr_interfaces = [self.cr_external]
```

Comment:
You've made it in opposite way - implemented half of the test add added a workaround which is extending the test whenever the bug is closed :)
The test should contain all the data by default and only in case bz is open data should be modified. Such way is easier to read, easier to maintain (as later on we can simply drop workaround without fixing the test) and basically logically correct. Please update PR